src/main/java/com/example/NiRi/service/OrderService.java

```diff
@@ -114,9 +114,12 @@ public Order createOrder(OrderCreationRequest orderRequest) {
         User user = userRepository.findById(orderRequest.getUserId())
                 .orElseThrow(() -> new RuntimeException("User not found with ID: " + orderRequest.getUserId()));
 
+        System.out.println("Order Creation Request: " + orderRequest);
```

Comment:
do we need these printlns still? could be a log.

Reply:
converted to logs